dockers/docker-orchagent/ipinip.json.j2

```diff
@@ -41,7 +41,6 @@
     {
         "TUNNEL_DECAP_TABLE:IPINIP_TUNNEL" : {
             "tunnel_type":"IPINIP",
-            "src_ip":"{{ ipv4_loopback_addresses | first | ip }}",
```

Comment:
You can remove the entire ipv4_loopback_addresses and ipv6_loopback_addresses arrays as well.